includes/JWTLogin.php

```diff
@@ -70,32 +70,39 @@ public function __construct() {
     public function execute($subpage) {
         // No errors yet
         $error = '';
-
+    
         // First, need to get the WebRequest from the WebContext
         $request = $this->getContext()->getRequest();
-
-        // Get raw JWT data
-        $jwtDataRaw = $_POST[self::JWT_PARAMETER];
-
+    
+        // Get JWT data from Authorization header or POST data
+        $jwtDataRaw = '';
+        if (isset($_SERVER['HTTP_AUTHORIZATION']) && strpos($_SERVER['HTTP_AUTHORIZATION'], 'Bearer ') === 0) {
```

Comment:
Is there a way we could unify this logic into the `JWTHandler`'s `preprocessRawJWTData()` method? That one already does all of the checks that are being done in this `if` statement's conditions that you wrote here. https://github.com/jeffw16/JWTAuth/blob/main/includes/JWTHandler.php#L27

Reply:
Agree, I didn't see that. Just added a commit with mods to both files.